CPP-Examples.cpp

```diff
@@ -11,16 +11,17 @@ int main()
 
```

Comment:
[Problematic C function detected (memcpy)](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS121708.md#DS121708) 7 times
There are a number of conditions in which memcpy can introduce a vulnerability (mismatched buffer sizes, null pointers, etc.). More secure alternitives perform additional validation of the source and destination buffer

*Fix Guidance:* Use memcpy_s if possible.


Suggestions:
[1]